js_modules/dagit/packages/core/src/nav/VersionNumber.tsx

```diff
@@ -1,23 +1,20 @@
-import {gql, useQuery} from '@apollo/client';
+import {useQuery} from '@apollo/client';
 import * as React from 'react';
 import styled from 'styled-components/macro';
 
-import {VersionNumberQuery} from './types/VersionNumberQuery';
+import {graphql} from '../graphql';
 
 export const VersionNumber = () => {
-  const {data} = useQuery<VersionNumberQuery>(VERSION_NUMBER_QUERY, {
-    fetchPolicy: 'cache-and-network',
-  });
-
+  const {data} = useQuery(VERSION_NUMBER_QUERY, {fetchPolicy: 'cache-and-network'});
   return <Version>{data?.version || <span>&nbsp;</span>}</Version>;
 };
 
 const Version = styled.div`
   font-size: 11px;
 `;
 
-const VERSION_NUMBER_QUERY = gql`
+const VERSION_NUMBER_QUERY = graphql(`
   query VersionNumberQuery {
     version
   }
-`;
+`);
```

Comment:
An example of what the change looks like. Note that we can remove the type annotations on the `useQuery` call above! 🎉